### ISSUE TITLE
Website down

### ISSUE THREAD

**Issue by SCKelemen** [vulkan-go/vulkan#16]
Has this project been abandoned? The website is currently down.
https://vulkango.com/

**Comment by SCKelemen** [comment]
Website is still down at https://vulkango.com

**Comment by xlab** [comment]
Yeah, the site was never been up, I currently have no time to make it. 😭

**Comment by SCKelemen** [comment]
@xlab, so is this library still supported?

**Comment by dragonfax** [comment]
bump.

**Comment by xlab** [comment]
> so is this library still supported?

@SCKelemen yes it is being maintained. The code was stable enough to do almost anything you want with Vulkan API, the 1.1 that was merged today as I found some time to verify stuff.

The problem with site is that I planned to create it, but then I found myself coding crypto projects for monies and my free time is currently occupied by basic life functions, I cannot create and maintain the site as a set of hand-crafted vulkan tutorials for Go, as I planned before. I will return to this idea hopefully in close future.

The package is maintained however.
